src/runtime_src/core/pcie/windows/alveo/device_windows.cpp

```diff
@@ -1054,8 +1055,19 @@ struct kds_cu_info
 
     auto& stats = (*it).second;
     result_type cuStats;
-    for (unsigned int i = 0; i < stats->CuCount; i++)
-      cuStats.push_back(std::make_tuple(stats->CuInfo[i].BaseAddress, stats->CuInfo[i].Usage, 0));
+    for (unsigned int i = 0; i < stats->CuCount; i++) {
```

Comment:
Did you test this on Windows?

Reply:
I don't know how to test windows. :( 
I am working with @pranjalv-xilinx and @arpitxilinx to make sure this doesn't break windows build.

Reply:
Yeah. Let's test it on windows before merging the code. Also, compare the old to new outputs to make sure all the information is correctly displayed